MAKE/Makefile.carrington_gcc_openmpi

```diff
@@ -62,7 +62,9 @@ FLAGS =
 #GNU flags:
 CC_BRAND = gcc
 CC_BRAND_VERSION = 9.4.0
-CXXFLAGS += -O3 -fopenmp -funroll-loops -std=c++17 -W -Wall -Wno-unused -mavx -march=znver2 #-flto
+CXXFLAGS += -O3 -fopenmp -funroll-loops -std=c++17 -mavx -march=znver2 #-flto
+CXXFLAGS += -Wall -Wextra -Wno-unused
+CXXFLAGS += -D OMPI_SKIP_MPICXX
 testpackage: CXXFLAGS = -O2 -fopenmp -funroll-loops -std=c++17 -mavx -march=znver2
```

Comment:
I would actually suggest that this testpackage line (which *overwrites* the CXXFLAGS values) should be placed before those warning-reducing lines, so that the warnings also get reduced for testpackage builds.